packages/repository/src/__tests__/integration/repositories/relation.factory.integration.ts

```diff
@@ -17,21 +15,31 @@ import {
   HasManyDefinition,
   HasManyRepository,
   HasManyRepositoryFactory,
-  juggler,
+  HasManyThroughDefinition,
+  HasManyThroughRepositoryFactory,
   ModelDefinition,
   RelationType,
+  createBelongsToAccessor,
+  createHasManyRepositoryFactory,
+  juggler,
 } from '../../..';
+import {
+  createHasManyThroughRepositoryFactory,
+  HasManyThroughRepository,
+} from '../../../relations';
```

Comment:
Is there any particular reason for importing from `../../../relations` instead of `../../..` as we do for most other types?

```suggestion
} from '../../..';
```